sqld/src/database/write_proxy/replication.rs

```diff
@@ -80,6 +80,7 @@ impl PeriodicDbUpdater {
 
 struct ReadReplicationHook {
     logger: WalLogClient<Channel>,
+    current_gen: u64,
```

Comment:
Same comment as https://github.com/libsql/sqld/pull/163#discussion_r1101303323

Reply:
those do not exist anymore